@AH-64D Apache Longbow/Addons/fza_ah64_hydra/functions/fn_eventFired.sqf

```diff
@@ -0,0 +1,28 @@
+/* ----------------------------------------------------------------------------
+Function: fza_fnc_eventFired
+
+Description:
+    Submunition creation event handler
+
+    projectile: Object
+    subMunitionProjectile: Object
+    pos: Array format PositionASL
+    velocity: Array format Vector3D
+
+Parameters:
+
+Returns:
+    Nothing
+
+Examples:
+
+Author:
+    Snow(Dryden)
+---------------------------------------------------------------------------- */
+params ["_projectile", "_submunitionProjectile", "_pos", "_velocity"];
+
+if (typeOf _submunitionProjectile == "fza_flec_cluster") then {
+    _counter = floor random 100;
+    if (_counter > 3) exitWith {};
```

Comment:
Has this been tested in multiplayer? My understanding was that `SubmunitionCreated` only ran on a single machine, and `drop` only has local effect